src/components/taxonomicService/taxonomicServiceFormComponents/FormBuilder.tsx

```diff
@@ -39,9 +47,20 @@ type Props = {
  * @returns JSX Component
  */
 const FormBuilder = (props: Props) => {
-    const { formTemplate } = props;
+    const { formTemplate, SetCompleted } = props;
+
+    /* Hooks */
+    const captchaHook = useCaptchaHook({
+        siteKey: import.meta.env.VITE_FRIENDLY_CAPTCHA_SITEKEY,
+        endpoint: "GLOBAL1",
```

Comment:
is the `1` a typo?

Reply:
Checked but its part of the endpoint: https://github.com/FriendlyCaptcha/friendly-captcha-react/blob/main/examples/react-example/src/App.tsx